database-emulator/javascript-quickstart/test/test.js

```diff
@@ -0,0 +1,115 @@
+let firebase = require("@firebase/testing");
```

Comment:
run a linter

Reply:
Do you have a favorite JS linter?

most require configs I can just use a generic one if you don't have suggestions.

Reply:
ESLint has a google style preset even though it's publically released. I'll use that.